Gems/Terrain/Code/Source/TerrainRenderer/EditorComponents/EditorTerrainMacroMaterialComponent.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <TerrainRenderer/EditorComponents/EditorTerrainMacroMaterialComponent.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/Serialization/EditContext.h>
+#include <AzToolsFramework/AssetBrowser/Entries/AssetBrowserEntry.h>
+
+namespace Terrain
+{
+    void EditorTerrainMacroMaterialComponent::Reflect(AZ::ReflectContext* context)
+    {
+        BaseClassType::ReflectSubClass<EditorTerrainMacroMaterialComponent, BaseClassType>(
+            context, 1,
+            &LmbrCentral::EditorWrappedComponentBaseVersionConverter<typename BaseClassType::WrappedComponentType,
+            typename BaseClassType::WrappedConfigType, 1>
```

Comment:
EditorWrappedComponentBaseVersionConverter is probably not necessary in O3DE at all. It was added when we implemented EditorWrappedComponentBase templates for dynamic vegetation to get rid of a lot of boilerplate editor component code. The version converter was necessary when injecting the new base class to update the serialized data to reflect the change. No new components need this and there is likely no data in O3DE that predates EditorWrappedComponentBase.